packages/portal/swagger-ui-web-component/package.json

```diff
@@ -17,7 +17,7 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "swagger-client": "^3.25.1",
-    "swagger-ui": "^4.19.1"
+    "swagger-ui": "5.1.0"
```

Comment:
question: is there a reason you're pinning the version?

Reply:
Unpinned